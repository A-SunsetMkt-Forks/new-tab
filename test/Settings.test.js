'use strict';

const Settings = require('../src/Settings.html');

describe('Settings root component', () => {
  it('renders correctly', () => {
    const target = document.createElement('div');
    new Settings({ target });
    expect(target.innerHTML).toMatchSnapshot();
  });
});
