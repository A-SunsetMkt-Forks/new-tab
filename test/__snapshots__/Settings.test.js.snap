// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Settings component renders correctly 1`] = `
"<div class=\\"row\\"><label for=\\"theme\\">Theme</label>
  <select id=\\"theme\\"><option value=\\"light\\">Light</option><option value=\\"null\\">Dark</option></select></div>

<hr>

<footer><small><em>â€» Settings sync to your account.</em></small></footer>"
`;
